### ISSUE TITLE
App Protection Policy Targeted Apps Missing

### ISSUE THREAD

**Issue by RJLucas123** [jseerden/IntuneBackupAndRestore#37]
When using the latest export module (v2.1.1) I noticed that the apps I am targeting with App Protection Policy do not get exported with the rest of the JSON.  If I use the PowerShell samples from Microsoft however, it exports just as it should.

**Comment by jseerden** [comment]
Hi @RJLucas123 , could you please include reproduction steps and perhaps a link to the PowerShell samples from Microsoft?

**Comment by RJLucas123** [comment]
Yes.  I run the following command: **Start-IntuneBackup -Path C:\IntunExport**.  The App Protection Policies are exported but if you open each JSON you see that the section called "apps" is completely missing.  Thus, if you import this JSON back into Intune the App Protection Policy is no longer targeting any applications.  When I use the ManagedAppPolicy_Export.ps1 from this sample site: "https://github.com/microsoftgraph/powershell-intune-samples/tree/master/AppProtectionPolicy" the export and JSON are correct.

**Comment by jseerden** [comment]
Hi @RJLucas123 could you please validate if this issue still exists with version 3.0.1 or later?
